frontend/src/routes/Search/Details/KubevirtPluginWrapper.tsx

```diff
@@ -2,18 +2,38 @@
 import React, { useContext, createContext } from 'react'
 import { PluginContext } from '../../../lib/PluginContext'
 import { KubevirtPluginData } from '../../../plugin-extensions/extensions/KubevirtContext'
+import { useAllClusters } from '../../Infrastructure/Clusters/ManagedClusters/components/useAllClusters'
```

Comment:
This is very expensive. I think you should just be able to look at the managed clusters themselves.